bootloader/src/pyi_archive.c

```diff
@@ -353,6 +356,8 @@ int pyi_arch_open(ARCHIVE_STATUS *status)
 		filelen = findDigitalSignature(status);
 		if (filelen < 1)
 			return -1;
+#else
```

Comment:
This *disables* searching the cookie for wind32 and apple. I assume, this should be `#endif`